.cirrus.yml

```diff
@@ -106,7 +106,7 @@ validate_task:
   eks_container:
     <<: *BUILDER_CONTAINER_DEFINITION
     cpu: 6
-    memory: 8G
+    memory: 10G
```

Comment:
I wonder how this works, the instance used is t3.small, which has 2GB of memory (and 2 vCPU)

Reply:
I will try to clarify that separately

Reply:
From what I observe in the graphics on the build, this value is used as we are now a bit below 10G in effective usage